app/assets/stylesheets/govuk-component/_related-items.scss

```diff
@@ -0,0 +1,24 @@
+.govuk-related-items {
```

Comment:
Fair point, happy to merge with the pixel values.

I think theres a discussion to have about consistency across components, but it's not a blocker for this PR :+1: